compiler/rustc_error_codes/src/error_codes/E0665.md

```diff
@@ -1,10 +1,9 @@
-#### Note: this error code is no longer emitted by the compiler.
-
-The `Default` trait was derived on an enum.
+The `Default` trait was derived on an enum without specifying the default
+variant.
 
 Erroneous code example:
 
-```compile_fail
+```compile_fail,ignore
```

Comment:
I had to do this to get over the stage0 test of this page.

Reply:
Doesn't the following make this work?

````md
```compile_fail,E0665
````